draft-ietf-quic-http.md

```diff
@@ -701,9 +708,9 @@ server to client, as in HTTP/2.  The PUSH_PROMISE frame defines no flags.
 The payload consists of:
 
 Push ID:
-: A 32-bit identifier for the server push request.  A push ID is used in push
-  stream header ({{server-push}}), CANCEL_PUSH frames ({{frame-cancel-push}}),
-  and PRIORITY frames ({{frame-priority}}).
+: A variable-length integer that identifies the server push request.  A push ID
```

Comment:
It'd be nice if we didn't have to call it a variable length integer everywhere.  The encoding is variable length, but I think of the integer being encoded as having a fixed length of 62 bits.  I'm not sure it's better to say a 62 bit integer or not.

Reply:
I think that this is OK.  XXX-bit and variable-length occupy the same mental space and the later is more correct.  Substitute 62-bit if that works for you, but I don't want to have to explain that 62-bit means the variable-length encoding.